lib/response.js

```diff
@@ -25,7 +25,7 @@ var sign = require('cookie-signature').sign;
 var normalizeType = require('./utils').normalizeType;
 var normalizeTypes = require('./utils').normalizeTypes;
 var setCharset = require('./utils').setCharset;
-var statusCodes = http.STATUS_CODES;
+var statusCodes = require('statuses/codes')
```

Comment:
I'm iffy about requiring any sub file in a module. The `statuses` API is lame, but probably can still be used here. I would just do `var statuses = require('statuses')` and then the code below would still work, after the variable name change (with a possible oddity in `res.sendStatus` that would need to be dealt with regardless of this change).